todoman/confspec.ini

```diff
@@ -51,3 +51,7 @@ startable = boolean(default=False)
 
 # When running ``todo`` with no commands, run this command.
 default_command = string(default='list')
+
+# The default priority of a task on creation.
+# Highest priority is 1, lowest priority is 10, and 0 means no priority at all.
+default_priority = integer(default=0)
```

Comment:
... And have this default to None.

This was if nothing is specified, we don't add a priority (which keeps the current default).